src/components/CytoscapeLayout/CytoscapeLayout.tsx

```diff
@@ -45,10 +46,7 @@ export default class CytoscapeLayout extends React.Component<CytoscapeLayoutProp
       elements: [],
       loading: false
     };
-    this.updateGraphElements = this.updateGraphElements.bind(this);
-    this.handleTap = this.handleTap.bind(this);
-    this.handleMouseIn = this.handleMouseIn.bind(this);
-    this.handleMouseOut = this.handleMouseOut.bind(this);
+    bindMethods(this, ['updateGraphElements', 'handleTap', 'handleMouseIn', 'handleMouseOut']);
```

Comment:
I'm -1 on this because you loose any compiler error in case of mistyping the function name.
Or could you put function reference in the array instead of strings?
`bindMethods(this, [this.updateGraphElements, this.handleTap, this.handleMouseIn, this.handleMouseOut]);`

Reply:
PS: I realize it may not be possible to use function references since `bind` doesn't modify the function passed as parameter, but return a new bound function instead